homeassistant/components/light/hue.py

```diff
@@ -174,7 +180,11 @@ def process_lights(hass, api, bridge, bridge_type, update_lights_cb):
             new_lights.append(bridge.lights[light_id])
         else:
             bridge.lights[light_id].info = info
```

Comment:
Maybe you can send this info with dispatcher

Reply:
Not sure, and that would definitely be a more ambitious change that doesn't really belong in this bug fix. @MartinHjelmare seemed to have ideas for further refactoring, that might fit in there.